ipyleaflet/leaflet.py

```diff
@@ -514,7 +514,7 @@ class GeoData(GeoJSON):
     geo_dataframe = Instance('geopandas.GeoDataFrame')
 
     def __init__(self, **kwargs):
-        super(GeoJSON, self).__init__(**kwargs)
+        super(GeoData, self).__init__(**kwargs)
```

Comment:
Good catch!! Is it why the on_hover was not working?

Reply:
yes definitely

Reply:
Now it works ! It deserves a tweet !  ;)